chat-plugins/info.js

```diff
@@ -14,7 +14,7 @@
 
 const RESULTS_MAX_LENGTH = 10;
 
-let commands = exports.commands = {
+let commands = exports.commands = { // eslint-disable-line no-unused-vars
```

Comment:
http://eslint.org/docs/rules/no-unused-vars#vars

Apparently setting it to `local` would work for this?

Although I think I'd prefer just getting rid of `let commands = `. ESLint _is_ right that we're not exactly using it for anything.